spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigMapPropertySource.java

```diff
@@ -64,37 +61,26 @@ protected static Environment createEnvironmentWithActiveProfiles(String[] active
 	}
 
 	protected static String getName(String name, String namespace) {
-		return new StringBuilder().append(PREFIX).append(Constants.PROPERTY_SOURCE_NAME_SEPARATOR).append(name)
-				.append(Constants.PROPERTY_SOURCE_NAME_SEPARATOR).append(namespace).toString();
+		return PREFIX + PROPERTY_SOURCE_NAME_SEPARATOR + name + PROPERTY_SOURCE_NAME_SEPARATOR + namespace;
 	}
 
 	protected static Map<String, Object> processAllEntries(Map<String, String> input, Environment environment) {
 
-		Set<Entry<String, String>> entrySet = input.entrySet();
+		Set<Map.Entry<String, String>> entrySet = input.entrySet();
 		if (entrySet.size() == 1) {
 			// we handle the case where the configmap contains a single "file"
-			// in this case we don't care what the name of t he file is
-			Entry<String, String> singleEntry = entrySet.iterator().next();
+			// in this case we don't care what the name of the file is
+			Map.Entry<String, String> singleEntry = entrySet.iterator().next();
 			String propertyName = singleEntry.getKey();
 			String propertyValue = singleEntry.getValue();
 			if (propertyName.endsWith(".yml") || propertyName.endsWith(".yaml")) {
-				if (LOG.isDebugEnabled()) {
```

Comment:
remove `isDebugEnabled`. I understand why it was done here - so that the message that is supposed to do in debug mode, is not computed unless debug is on. But the price to pay for a branch is usually much higher than an allocation. 

Overall, I blame the library itself, for not providing a `Supplier<String>` as an argument to `LOG::debug`...